pkgs/tools/graphics/waifu2x-converter-cpp/default.nix

```diff
@@ -1,34 +1,64 @@
-{ cmake, fetchFromGitHub, opencv3, stdenv, opencl-headers
-, cudaSupport ? false, cudatoolkit ? null
+{ stdenv
+, fetchFromGitHub
+, makeWrapper
+
+, cmake
+, cudatoolkit
+, ocl-icd
+, opencl-headers
+, opencv3
+, pkgconfig
+, waifu2x-converter-cpp
```

Comment:
Wouldn't it be better to pass `doCheck` as an option instead of this kind of self reference?
Like here: https://github.com/NixOS/nixpkgs/blob/157d56d311aacbbcc8191a7c6ec0fda155b5fea9/pkgs/development/libraries/glib/default.nix#L8

Reply:
Yeah, that seems a lot simpler!